frontend/src/components/common/Game/AdminMenu.vue

```diff
@@ -89,6 +99,12 @@ async function switchFromFavourites() {
 
 <template>
   <v-list rounded="0" class="pa-0">
+    <v-list-item class="py-4 pr-5" @click="openInNewTab()">
+      <v-list-item-title class="d-flex">
+        <v-icon icon="mdi-open-in-new" class="mr-2" />Open in a new tab
+      </v-list-item-title>
+    </v-list-item>
+    <v-divider />
```

Comment:
Can you remove this change and keep all the others? I like the other tweaks overall 👍🏼